pages/components/DataTable.neu.page.ts

```diff
@@ -0,0 +1,85 @@
+import { expect, type Locator, Page } from '@playwright/test';
+
+export class DataTable {
+    /* since the table is within Vuetify's jurisdiction,
+        we cannot specify test ids for Playwright and heavily rely on classes as locators */
+    readonly page: Page;
+    readonly tableLocator: Locator;
+
+    constructor(page: Page, locator: Locator) {
+        this.page = page;
+        this.tableLocator = locator;
+    }
+
+    /* actions */
+    public getItemByText(expectedText: string): Locator {
+      return this.tableLocator.getByRole('cell', { name: expectedText, exact: true });
+    }
+
+    public setItemsPerPage(value: string): void {
+      const itemsPerPageSelect: Locator = this.page.locator('.v-data-table-footer__items-per-page .v-select');
+
+      itemsPerPageSelect.selectOption(value);
+    }
+
+    public async goToFirstPage(): Promise<void> {
+      const goToFirstPageButton: Locator = this.page.locator('.v-pagination__first button:not(.v-btn--disabled)');
+      
+      await goToFirstPageButton.click();
+    }
```

Comment:
```suggestion
  public async goToFirstPage(): Promise<void> {
    await this.page.locator('.v-pagination__first button:not(.v-btn--disabled)').click();
  }
```
Just inline the whole expression since its only used once. You can do this to all other methods here.